docs/technical_reference/flowsheets/electrodialysis_1stack_conc_recirc.rst

```diff
@@ -0,0 +1,116 @@
+One-stack electrodialysis with a concentrate fluid recirculation
```

Comment:
Just for consistency with the Other "One-Stack Electrodialysis" and other fs titles
```suggestion
One-Stack Electrodialysis with a Concentrate Fluid Recirculation
```